core/new-gui/src/app/dashboard/user/component/user-workflow/ngbd-modal-workflow-executions/workflow-execution-modal.component.ts

```diff
@@ -21,7 +22,7 @@ const MAX_USERNAME_SIZE = 5;
   styleUrls: ["./workflow-execution-modal.component.scss"],
 })
 export class WorkflowExecutionModalComponent implements OnInit, AfterViewInit {
```

Comment:
Rename the component since it's no longer a modal.

Reply:
Done